src/pages/TodoPage/TodoPageContainer.tsx

```diff
@@ -21,14 +27,28 @@ export const TodoPageContainer: React.FC = () => {
   const [visibleModal, setVisibleModal] = useState(false);
   const [initialValue, setInitialValue] = useState(defaultValue as ITodo);
 
+  const fetchData = async (url: string) => {
+    const { data } = await axios.get<ITodo[]>(url);
+    setTodos(data);
+  };
+
+  useEffect(() => {
+    fetchData('http://localhost:4000/api');
+  }, []);
+
   const hasTodo = !!filteredTodos.length;
 
-  const addTodo = (todo: IPureTodo) => {
-    setTodos([...todos, { ...todo, id: v4() }]);
+  const addTodo = async (todo: IPureTodo) => {
+    const { data } = await axios.post('http://localhost:4000/api/', todo);
```

Comment:
Create a helper. Store it in a separate file.

Reply:
done